packages/language-server/src/server.ts

```diff
@@ -322,7 +322,7 @@ export function startServer(options?: LSOptions) {
                 },
                 documentHighlightProvider:
                     evt.initializationOptions?.configuration?.svelte.plugin.svelte
-                        ?.documentHighlight?.enable === false
+                        ?.documentHighlight?.enable === true
```

Comment:
Oops. Got it wrong. What I actually wanted is `evt.initializationOptions?.configuration?.svelte.plugin.svelte?.enable !== false`. But now that I think about it `evt.initializationOptions?.configuration?.svelte.plugin.svelte?.enable ?? true` is probably clearer.